src/wasm-binary.h

```diff
@@ -261,17 +261,32 @@ enum ASTNodes {
   F64Le = 0x9a,
   F64Gt = 0x9b,
   F64Ge = 0x9c,
```

Comment:
The ordering is just a historical accident of how @titzer added the opcodes to the implementation, IIRC.